src/node/desktop/package.json

```diff
@@ -71,6 +71,7 @@
     "i18next": "21.8.13",
     "line-reader": "0.4.0",
     "lodash.debounce": "4.0.8",
+    "net-ipc": "2.0.1",
```

Comment:
Please add this to the NOTICE file. There should be an entry in the list near the top of the file, and the license details later in the file.